tools/PI/DevHome.PI/Pages/WERPage.xaml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page
+    x:Class="DevHome.PI.Pages.WERPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
+    xmlns:models="using:DevHome.PI.Models"
+    mc:Ignorable="d"
+    NavigationCacheMode="Enabled">
+
+    <Grid>
+        <Grid.Resources>
+            <Style TargetType="TextBlock">
+                <Setter Property="Margin" Value="0,0,0,6"/>
+            </Style>
+        </Grid.Resources>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="2*"/>
+            <RowDefinition Height="3*"/>
+        </Grid.RowDefinitions>
+
+        <StackPanel Orientation="Vertical">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="Auto" />
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <TextBlock x:Uid="WERHeaderTextBlock" FontSize="{StaticResource SubtitleTextBlockFontSize}" FontWeight="SemiBold" Margin="0,0,0,8" Grid.ColumnSpan="2"/>
+
+                <StackPanel x:Name="AppLocalDumpCollection" Visibility="{x:Bind ViewModel.AttachedToApp, Mode=OneWay}" Orientation="Horizontal" Spacing="5" Grid.Row="1" Grid.Column="1">
+                    <TextBlock x:Uid="LocalDumpCollectionLabel" VerticalAlignment="Center" Margin="0 0 10 0"/>
+                    <ToggleSwitch x:Uid="LocalDumpCollectionToggle" x:Name="LocalDumpCollectionToggle" IsOn="{x:Bind ViewModel.LocalCollectionEnabledForApp, Mode=TwoWay}" IsEnabled="{x:Bind ViewModel.RunningAsAdmin}" Toggled="LocalDumpCollection_Toggled" Margin="0 0 -70 0" />
```

Comment:
A negative Margin usually gives me pause. What's happening here?

Reply:
Rightfully so. The toggle switch seems to give a lot of space for it's on/off label. I'm sure this will blow up in my face when it comes to localization, so I'm hoping to get this officially designed by Design before this becomes an issue.

![image](https://github.com/microsoft/devhome/assets/49733346/225d3594-467f-4df8-8a61-14a88fef4bb0)

The toggle on the bottom is the standard spacing/margins for a toggle... see the space between the "Off" and the Settings icon?

I did not want that spacing for the top toggle, hence the pretty big negative margin. Design is thinking about what an elevation UX should look like, so I expect this to churn in the future.

Reply:
Looks like DefaultToggleSwitchStyle's MinWidth == ToggleSwitchThemeMinWidth == 154, which seems like a lot. We can either leave this for now, or you could probably set the MinWidth to something smaller. I think that would look like this:
```suggestion
                    <ToggleSwitch x:Uid="LocalDumpCollectionToggle" x:Name="LocalDumpCollectionToggle" IsOn="{x:Bind ViewModel.LocalCollectionEnabledForApp, Mode=TwoWay}" IsEnabled="{x:Bind ViewModel.RunningAsAdmin}" Toggled="LocalDumpCollection_Toggled" Margin="0 0 -70 0">
                        <ToggleSwitch.Resources>
                            <x:Double x:Key="ToggleSwitchThemeMinWidth">50</x:Double>
                        </ToggleSwitch.Resources>
                    </ToggleSwitch>
```

Reply:
The MinWidth="0" should make it so localization doesn't cause me problems.